docs/tasks/exporting-and-importing.md

```diff
@@ -1,88 +1,187 @@
-# Exporting and Importing in Nuclio
+# Exporting and Importing Nuclio Functions and Projects
```

Comment:
Why naming resources?

it export and import apigateway as well, and function events

Reply:
I'm not aware of an option to export and import API gateways or function events (outside of a project import). The doc and help text before my edits referred only to exporting and importing functions and projects (including functions and function events). Also, I only see  `functions` and `projects` import and export CLI commands. => ?

Reply:
There's no explicit option to export only apigateway / function events as per this moment - mainly because there's no demand for that.

What I mean here, is that we can more generic on the title, and provide more details along the doc.
for the record, when you export project, you export apigateway and function events along with the project and functions.

Reply:
@liranbg (a) I can can change the title to "Importing and Exporting Resources".
(b) We already documented (in the doc and help text) that projects exports and imports also export and import all of the project's functions and function events. Should I add API gateways to this? Can a project have more than one API gateway?

Reply:
A) Great
B) yes and yes.

Reply:
a) DONE — I edited the title in commit 630951d6.
b) DONE — I edited the doc & `export`/`import` help to refer also to API gateways; see commit 52a1250.